src/fides/api/graph/config.py

```diff
@@ -443,6 +444,9 @@ class MaskingOverride:
     length: Optional[int]
 
 
+PARTITION_CLAUSE_PATTERN = r"`(?P<field_1>[a-zA-Z0-9_]*)` ([<|>][=]?) ([a-zA-Z0-9_\s(),.]*) AND `(?P<field_2>[a-zA-Z0-9_]*)` ([<|>][=]?) ([a-zA-Z0-9_\s(),.]*)$"
```

Comment:
is any of this bigquery specific? if so we might note that

Reply:
yeah good call, i think the back ticks may be BQ-specific, and we're already scoped down to BQ in other respects so let me just make that more explicit 👍